README.md

```diff
@@ -8,8 +8,20 @@
 
 ### Demo
 
+Installation & Help message
+[![asciicast](https://asciinema.org/a/616538.svg)](https://asciinema.org/a/616538)
+
+Plain JSON files diff:
 [![asciicast](https://asciinema.org/a/602583.svg)](https://asciinema.org/a/602583)
+
+Plain YAML files diff:
 [![asciicast](https://asciinema.org/a/604893.svg)](https://asciinema.org/a/604893)
```

Comment:
this asciinema has JSON and yaml file format, so you can use it instead of separate asciinema for JSON file (don't forget to add new header, smth like `Plain JSON and YAML files diff`) (or record a new one for yaml files only)